static/src/stylesheets/module/facia-garnett/_item.scss

```diff
@@ -193,6 +193,8 @@ $fc-item-gutter: $gs-gutter / 4;
 
 .fc-item__header {
     flex: 1 1 auto;
+    //this should eventually get added to the __src.scss
```

Comment:
this will never get added to `__src.scss` :) if it stays here, how does impact things?

Reply:
Cool. Yeah I realise now that that is stupid!

Reply:
Being placed here it overrides font-weight:500 which is defined elsewhere, hence the confusion with it being place in src.